src/astro/constants.py

```diff
@@ -1,2 +1,5 @@
+import os
+
 DEFAULT_CHUNK_SIZE = 1000000
 PYPI_PROJECT_NAME = "astro-projects"
+DEFAULT_SCHEMA = os.getenv("AIRFLOW__ASTRO__SQL_SCHEMA") or "astroflow_ci"
```

Comment:
I feel that we should not be defaulting the schema to `astroflow_ci` in the application - this should only happen in the test context. The previous value `tmp_astro` seemed to make more sense from a user perspective.

I also believe `constants.py` should contain constants - and not user-defined values through environment variables. We could have `settings.py` or something similar for this value type.